api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/dashboards_controller.rb

```diff
@@ -79,21 +90,19 @@ def copy
   private
 
     def dashboard(*included)
-      # TODO: [APIv2] Improve filtering by owner (owner_type?)
-      @dashboard ||= MnoEnterprise::Dashboard.where(owner_id: current_user.id).includes(included).find(params[:id].to_i).first
+      @dashboard ||= MnoEnterprise::Dashboard.includes(included).find(params[:id]).first
```

Comment:
Security issue :scream: 
If you're removing the scoping, you should then look at implementing the `TODO: authorization` above

Reply:
No need to, as it's checked later in the authorize.

Reply:
This one?
```ruby
# TODO: enable authorization
# authorize! :manage_dashboard, dashboard
```